src/pages/en/tutorial/3-components/4.md

```diff
@@ -45,8 +45,8 @@ Create a `<Hamburger />` component to open and close your mobile menu.
 
 ```astro title="src/components/Header.astro" ins={2,7}
 ---
-import Hamburger from '../components/Hamburger.astro';
-import Navigation from '../components/Navigation.astro';
+import Hamburger from './components/Hamburger.astro';
+import Navigation from './components/Navigation.astro';
```

Comment:
```suggestion
import Hamburger from './Hamburger.astro';
import Navigation from './Navigation.astro';
```
Thanks for jumping on this @khalatevarun! It's very helpful when I am making a bunch of issues at once when I don't have to then go back and fix them.  😄 

In fact, the import statement gets even shorter because we're importing components from the same folder (`/components/`) so all that's needed is the `./`

I'll just commit this change because it's quick, so I can officially welcome you to Team Docs! 🥳